src/twisted/internet/defer.py

```diff
@@ -159,6 +165,65 @@ def maybeDeferred(f, *args, **kw):
         return succeed(result)
 
 
+def unbox(deferred, raises=True, passthrough=True):
+    """
+    Unbox a L{Deferred} that has already been fully processed.
+
+    Take an already processed (i.e. no longer waiting on any callbacks to be
+    fired) and unbox it. If the result is a L{Failure}, then it will re-raise
+    the original exception, otherwise it will return the result of the
+    L{Deferred}.
+
+    B{This function CANNOT take a Deferred with pending asynchronous work to be
+    done and make it synchronous. It can ONLY unbox a Deferred which already
+    has successfully completed.}
+
```

Comment:
I think even if you don't borrow the existing terminology that at least linking to `successResultOf` might be a nice crosslink.